src/fileLoader.js

```diff
@@ -17,7 +17,7 @@ module.exports = function( relativeFile ) {
 
 	let result = cache[ fileName ];
 	if ( !result ) {
-		result = cache[ fileName ] = readFileAsync( fileName, "utf8" );
+		result = cache[ fileName ] = await readFileAsync( fileName, "utf8" );
```

Comment:
I know this is not part of this PR, but this cache looks like a great place for a `Map`.